mirrord/cli/src/execution.rs

```diff
@@ -289,10 +289,9 @@ impl MirrordExecution {
             })?;
 
         // Provide details for layer to connect to agent via internal proxy
-        env_vars.insert(
-            MIRRORD_CONNECT_TCP_ENV.to_string(),
-            format!("127.0.0.1:{}", address.port()),
-        );
+        let mut config = config.clone();
```

Comment:
Instead of `clone` to `mut` here, why not take this as `&mut` in the function args?